ssl/statem/statem.h

```diff
@@ -86,10 +84,17 @@ struct ossl_statem_st {
     READ_STATE read_state;
     WORK_STATE read_state_work;
     OSSL_HANDSHAKE_STATE hand_state;
+    /* The handshake state requested by an API call (e.g. HelloRequest) */
+    OSSL_HANDSHAKE_STATE request_state;
```

Comment:
I'm not sure. I remember that during the state machine review I was asked to change a number of things to have the OSSL_ prefix, but I don't recall if this was one of them.